electrolytes/runsolvent.py

```diff
@@ -0,0 +1,26 @@
+from openmm.app import *
+from openmm import *
+from openmm.unit import *
+from sys import stdout, exit
+
+
+#TO-DO: Read temperature from CSV file
```

Comment:
Still need to be done? Seems like temp is fixed at 293 for now.

Reply:
Yeah. It's a simple matter of reading from the previous CSV file, I've just been putting off editing this part of the code.